src/encryption/api.go

```diff
@@ -0,0 +1,88 @@
+package encryption
+
+import (
+	"context"
+	"fmt"
+	"keboola-as-code/src/client"
+	"keboola-as-code/src/remote"
+
+	"github.com/go-resty/resty/v2"
+	"go.uber.org/zap"
+)
+
+type Api struct {
+	apiHostUrl string
+	client     *client.Client
+	logger     *zap.SugaredLogger
+}
+
+// Error represents Encryption API error structure
+type Error struct {
+	Message     string `json:"error"`
+	ErrCode     int    `json:"code"`
+	ExceptionId string `json:"exceptionId"`
+}
+
+func (e *Error) Error() string {
+	msg := fmt.Sprintf(`"%v", errCode: "%v"`, e.Message, e.ErrCode)
+	if len(e.ExceptionId) > 0 {
+		msg += fmt.Sprintf(`, exceptionId: "%s"`, e.ExceptionId)
+	}
+	return msg
+}
+
+func getEncryptionApiHost(connectionApiHost string, ctx context.Context, logger *zap.SugaredLogger) string {
+	storageApi := remote.NewStorageApi(connectionApiHost, ctx, logger, false)
+	services, err := storageApi.GetServices()
```

Comment:
Okrem `GetServices`, by som API vytvoril aj metodu `GetEncryptionApiHost` a presunul to tam.
Tym padom nebudes mat medzi balickom `encryption` a `remote` ziadnu priamu zavislost, co je cistejsie.
Bude to prepojene iba cez `host string`.
A rovno v `remote` balicku dava zmysel mat aj test na `GetEncryptionApiHost`.